docs/chapter_sorting/radix_sort.md

```diff
@@ -0,0 +1,190 @@
+---
+comments: true
+---
+
+# 基数排序
+
+「基数排序 Radix Sort」是基于**分桶**思想实现的**非比较型**稳定的排序算法，是对**桶排序的扩展**。
+
+!!! question "基数排序与桶排序的区别在哪？"
+
+    基数排序对要被排序的数据是有要求的，这些数据需可以被分割出独立的"位"，并且位与位之间要有递进关系: 如果a数据的高位比b数据大，
+    那么剩下的低位就不再需要比较了。而桶排序的核心思想是先定义几个有序的 "桶"，将要排序的数据分到这几个 "桶" 里，
+    再对每个 "桶" 里的数据单独进行排序， 之后将 "桶" 里的数据按顺序取出，即可获得排序后的结果值，看到这里这两个排序好像没有什么区别，
+    但是桶排序并不需要被排序的数据满足位与位有递进关系的要求。
+
```

Comment:
We should introduce how the Radix Sort works before describing the algorithm process.